nl_server/embeddings.py

```diff
@@ -22,11 +24,60 @@
 from sentence_transformers.util import semantic_search
 import torch
 
+from nl_server import query_util
 import nl_server.gcs as gcs
+from server.lib.nl import utils
 
 TEMP_DIR = '/tmp/'
 MODEL_NAME = 'all-MiniLM-L6-v2'
 
+# A value higher than the highest score.
+_INIT_SCORE = 1.1
```

Comment:
Maybe have _HIGHEST_SCORE and use (_HIGHEST_SCORE + 0.1) for _INIT_SCORE so the purpose is more apparent?